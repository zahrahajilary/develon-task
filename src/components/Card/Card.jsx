import React from "react";
import { CardContainer } from "./Card.styled";

const Card = ({ imgUrl, index }) => {
  return (
    <CardContainer key={index}>
      <img src={`${imgUrl}`} alt="cat-picture" />
    </CardContainer>
  );
};
export default Card;
