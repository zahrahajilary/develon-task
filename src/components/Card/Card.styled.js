import styled from "styled-components";

export const CardContainer = styled.div`
  padding: 16px;
  margin: 16px;
  background-color: whitesmoke;
  border-radius: 16px;
  border: 1px solid lightgray;
  width: 16rem;
  height: 16rem ;
  justify-content: center;
`;

export const ImgContainer = styled.img``;
