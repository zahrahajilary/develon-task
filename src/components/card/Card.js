import React from 'react';


const Card = ({imgUrl,index,title})=> {
    return(<div>
        <div style={{border: '1px solid white', padding: 16, margin: 16,flexGrow:1}} key={index}>
            <img src={`${imgUrl}`} alt={title} width={250} height={250}/>
        </div>
    </div>)
}
export default Card
