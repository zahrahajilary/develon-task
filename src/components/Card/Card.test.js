
import { shallow } from 'enzyme';
import React from 'react';

import Card from './Card';

it('should render Card component', () => {

    expect(shallow(<Card imgUrl='assets' index={1}/>)).toMatchSnapshot();
});
