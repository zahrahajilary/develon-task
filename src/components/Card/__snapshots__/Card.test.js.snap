// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Card component 1`] = `ShallowWrapper {}`;
